engine/src/main/java/org/entando/entando/aps/system/services/page/PageTokenManager.java

```diff
@@ -102,11 +99,9 @@ public String decrypt(String property) {
 			Cipher pbeCipher = Cipher.getInstance("PBEWithMD5AndDES");
 			pbeCipher.init(Cipher.DECRYPT_MODE, key, new PBEParameterSpec(this.getSalt().getBytes(), 20));
 			return new String(pbeCipher.doFinal(base64Decode(property)), "UTF-8");
-		} catch (GeneralSecurityException e) {
-			logger.error("Error in decrypt");
-		} catch (IOException e) {
+		} catch (GeneralSecurityException | IOException e) {
 			logger.error("Error in decrypt");
```

Comment:
If we're changing this we should also log the exception itself `logger.error("Error in decrypt",e);

Reply:
Fixed